dev-support/pysubmarine/gen-sdk.sh

```diff
@@ -55,13 +61,15 @@ java -jar ${SWAGGER_CODEGEN_JAR} generate \
      -c ${SWAGGER_CODEGEN_CONF}
 
 echo "Insert apache license at the top of file ..."
-for filename in $(find ${SDK_OUTPUT_PATH}/submarine/experiment -type f); do
+for filename in $(find ${SDK_OUTPUT_PATH}/submarine/${API_COMPONENT} -type f); do
   echo "$filename"
-  sed -i -e '1 e cat license-header.txt' "$filename"
```

Comment:
I give up using `sed` since there is some differences between the usage of BSD and Linux version.